assembler/patterns/byline-date.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Title: Byline and Date Post Meta
+ * Slug: assembler/byline-date
+ * Inserter: no
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"blockGap":"0.26rem","padding":{"bottom":"20px"}},"typography":{"fontSize":"x-small"}},"layout":{"type":"flex","flexWrap":"nowrap","verticalAlignment":"center","justifyContent":"center"}} -->
+<div class="wp-block-group" style="padding-bottom:20px;font-size:x-small"><!-- wp:paragraph {"fontSize":"x-small"} -->
+<p class="has-x-small-font-size"><?php echo esc_html_x( 'Published by', 'assembler' ); ?></p>
```

Comment:
> [!TIP]
> When we are not passing context information for the translators, we should use [esc_html_e](https://developer.wordpress.org/reference/functions/esc_html_e/) rather than [esc_html_x](https://developer.wordpress.org/reference/functions/esc_html_x).
>
> Otherwise, `esc_html_x` uses "assembler" as the context information for the translators, and "default" as the domain when "assembler" should be the domain. The domain is a unique identifier for retrieving translated strings.

Reply:
Done. TIL. I just copied from Creatio 2 😂